src/app/shared/code-mirror/code-mirror.component.ts

```diff
@@ -32,11 +33,30 @@ export class Class implements AfterContentInit, OnChanges, OnDestroy {
 	@Input() codeMirror:CodeMirror.Editor;
 	@Output() codeMirrorChange:EventEmitter<CodeMirror.Editor> = new EventEmitter<CodeMirror.Editor>();
 
+	//TODO: this is a shortcut that could be enable to prettify, at this moment we decide disabled.
+	/*@HostListener( 'keydown', [ '$event' ] ) onKeyDown( e ) {
```

Comment:
To preserve code cleanliness, unused functionality should be removed from code (instead of being left behind as comments